contents/python/introduction-to-python.md

```diff
@@ -4,25 +4,30 @@ Python is a simple yet powerful and versatile language. Conceived in the late 80
 
 ## Getting started with Python
 
-[Google's Python Class](https://developers.google.com/edu/python/)
+If you're reading this on GitHub, I'm guessing that you're already an experienced programmer looking to get in on the Python action. If that's the case, check out [Google's Python class](https://developers.google.com/edu/python/), which will introduce you to Python's concepts and core data structures like lists, dictionaries, and strings!
+
+For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N4mEzFDjqtA) by Derek Banas where he covers everything from installing Python and the basics to Inheritance and Polymorphism(!!!) in under an hour! If you'd prefer to read, check out [Python Guru](http://thepythonguru.com/) which has plenty of code samples to help you along.
```

Comment:
Hm it was more like to underscore the amount of content he covered in under an hour, with the basics (installing) at one end and the "advanced" concepts at the other, but it seems like it wasn't obvious. Any suggestions on improving this, or should I just remove it altogether?